### PR TITLE
cleanup of user api

### PR DESCRIPTION
**Pull request by alichaddad**
https://github.com/0-complexity/openvcloud/issues/955

### CODE REVIEW

File: apps/cloudbroker/base/cloudapi__users/methodclass/cloudapi_users.py [hunk 1]
```diff
@@ -21,32 +21,6 @@ def __init__(self):
         self.acl = j.clients.agentcontroller.get()
         self.systemodel = j.clients.osis.getNamespace('system')
 
-    def authenticate(self, username, password, **kwargs):
```
Comment (grimpy) [inline apps/cloudbroker/base/cloudapi__users/methodclass/cloudapi_users.py]: I thought we agreed to keep this one around?

File: apps/cloudbroker/base/cloudapi__users/specs/actors.spec [hunk 2]
```diff
@@ -3,17 +3,6 @@
 	User management
 	"""
 
-	method:authenticate @noauth
```
Comment (grimpy) [inline apps/cloudbroker/base/cloudapi__users/specs/actors.spec]: lets keep it?

**Comment by grimpy** [comment]
After some discussion with @rkhamis and @yveskerwyn we decided to keep authenticate API for a while

File: apps/cloudbroker/base/cloudapi__users/specs/actors.spec [hunk 3]
```diff
@@ -3,7 +3,7 @@
...
+    method:authenticate @noauth
```
Comment (grimpy) [inline apps/cloudbroker/base/cloudapi__users/specs/actors.spec]: changes in whitespaces? You sure its still loads? Actor spec parser is very picky about these kind of things

Comment (alichaddad) [reply apps/cloudbroker/base/cloudapi__users/specs/actors.spec]: Yes, it is more consistent with the rest of the methods in the spec as well.

**Pull request merged**